frontend/src/pages/CalendarPage/components/EventCard/index.js

```diff
@@ -10,10 +10,18 @@ import styles from './styles';
 
 import { Card, GetLocation } from '@SharedComponents';
 
-function EventCard({ event, classes }) {
-  return (
-    <>
-      {event && (
+function EventCard({ event, onClose, classes }) {
+  // Listens to whether an event is selected or not from Calendar parent comp
+  const isOpen = event ? true : false;
```

Comment:
can we do event === null instead of the ternary? Javascript Boolean coercion is not that straightforward :p